### PR TITLE
upgrade go to 1.13.7

### PR DESCRIPTION
**Pull request by ggilmore**

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,2 +1,2 @@
-golang 1.13.5
+golang 1.13.7
```
Comment (creachadair) [inline .tool-versions]: Have the images on Dockerhub been updated yet? (Or are we using a different registry that has?)

Comment (ggilmore) [reply .tool-versions]: Which docker images?

Comment (creachadair) [reply .tool-versions]: I was assuming these are meant to be image tags, based on the use of `golang`. If not, maybe that is irrelevant.

Comment (ggilmore) [reply .tool-versions]: No, this is a file that tells CI which versions of the `go`, `yarn` binaries to use when running our pipeline. See https://asdf-vm.com/

**Pull request merged**